projects/external-components/src/lib/primeng-selectors/components/iframe-external/iframe-external.component.html

```diff
@@ -1,6 +1,9 @@
-
-<div>
-    <iframe [src]="src" style="height:90%; width:100%">  
+<div *ngIf= "visible" style="height:88vh">
```

Comment:
We need to make the height and width of iframe configurable.

Reply:
currently it is full page